packages/config/config/devices/0x0346/alarm_keypad_gen1.json

```
@@ -344,55 +343,55 @@
 				}
 			]
 		},
-		"23[0x1]": {
+		"23[0x1000000]": {
```

Please use two digits per byte in the mask, even if that introduces a leading 0 - makes it easier to grasp.
```suggestion
		"23[0x01000000]": {
```

---

same for the others